### PR TITLE
Issue 372 - Icon color not updating with theme color

### PR DESCRIPTION
**Pull request by igloo12**
Addresses Issue https://github.com/kafbat/kafka-ui/issues/372

Screenshot 2024-06-21 at 4 55 36 PM
Screenshot 2024-06-21 at 4 54 35 PM

- [ ] **Breaking change?** (if so, please describe the impact and migration path for existing application instances)

**What changes did you make?** (Give an overview)
Updated the icon svgs to use the theme
**Is there anything you'd like reviewers to focus on?**

**How Has This Been Tested?** (put an "x" (case-sensitive!) next to an item)

- [ ] No need to
- [x] Manually (please, describe, if necessary)
- [ ] Unit checks
- [ ] Integration checks
- [ ] Covered by existing automation

**Checklist** (put an "x" (case-sensitive!) next to all the items, otherwise the build will fail)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (e.g. **ENVIRONMENT VARIABLES**)
- [x] My changes generate no new warnings (e.g. Sonar is happy)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged

Check out [Contributing](https://github.com/kafbat/kafka-ui/blob/main/.github/CONTRIBUTING.md) and [Code of Conduct](https://github.com/kafbat/kafka-ui/blob/main/.github/CODE-OF-CONDUCT.md)

**A picture of a cute animal (not mandatory but encouraged)**

Red-Panda-Image-for-Endangered-Excursion-scaled

### CODE REVIEW

**Review by Nilumilak — COMMENTED** [review]
I think it would be better to connect global theme to the colors instead of making the svg transparent

**Comment by Nilumilak** [comment]
> > I think it would be better to connect global theme to the colors instead of making the svg transparent
> 
> Isn't that what the `styled()` method does? I copied it from the edit icon. The `styled()` sets the fill color of the svg.

edit icon uses the global  theme for filling the svg
```tsx
fill={theme.icons.editIcon.normal}
```

File: frontend/src/components/common/Icons/CheckmarkIcon.tsx [hunk 1]
```diff
@@ -13,18 +14,9 @@ const CheckmarkIcon: FC = () => {
...
-        data-name="layer1"
-        fill="none"
-        stroke="#FFFFFF"
-        strokeMiterlimit="10"
-        strokeWidth="2"
-        d="M2 30l21 22 39-40"
-        strokeLinejoin="round"
-        strokeLinecap="round"
-      />
+      <path d="M25 48c-1.1 0-2.2-.4-3-1.2l-13-13c-1.6-1.6-1.6-4.2 0-5.8s4.2-1.6 5.8 0L25 38.8 49.2 14.6c1.6-1.6 4.2-1.6 5.8 0s1.6 4.2 0 5.8l-27 27c-.8.8-1.9 1.2-3 1.2z" />
```
Comment (Haarolean) [inline frontend/src/components/common/Icons/CheckmarkIcon.tsx:17]: What's happening here, why is the shape changed?

File: frontend/src/components/common/Icons/CancelIcon.tsx [hunk 2]
```diff
@@ -16,10 +15,7 @@ const CancelIcon: React.FC = () => {
       <title>Cancel</title>
       <desc>A line styled icon from Orion Icon Library.</desc>
       <path
-        data-name="layer1"
         d="M53.122 48.88L36.243 32l16.878-16.878a3 3 0 0 0-4.242-4.242L32 27.758l-16.878-16.88a3 3 0 0 0-4.243 4.243l16.878 16.88-16.88 16.88a3 3 0 0 0 4.243 4.241L32 36.243l16.878 16.88a3 3 0 0 0 4.244-4.243z"
-        fill="none"
-        stroke={theme.icons.cancelIcon}
```
Comment (Haarolean) [inline frontend/src/components/common/Icons/CancelIcon.tsx:22]: the color should be different for two themes, how is this supposed to work after the changes you've introduced?

**Review by Haarolean — CHANGES_REQUESTED** [review]

**Pull request closed without merging**